tests/shaping/main.rs

```diff
@@ -92,7 +116,15 @@ pub fn shape(font_path: &str, text: &str, options: &str) -> String {
     }
 
     let mut buffer = rustybuzz::UnicodeBuffer::new();
+    if let Some(pre_context) = args.pre_context {
+        eprintln!("precontext: {pre_context:#?}");
```

Comment:
Why do we need to print it?

Reply:
Forgot to remove it after debugging a problem.